common/gratia/common/condor_ce.py

```diff
@@ -359,7 +359,7 @@ def queryJob(jobid):
         job_info = queryAllJobs()
         for classad in job_info.values():
             # On failure, there is not much to do - ignore
-            DebugPrint("Creating certinfo file for %s." % \
+            DebugPrint(3, "Creating certinfo file for %s." % \
```

Comment:
While we're here ............

Nbd either way, but `\` line continuators are a bit of an eyesore when they are unnecessary (eg here inside parens)

```suggestion
            DebugPrint(3, "Creating certinfo file for %s." %
```